Blankmon/src/application/moves/Tackle.java

```diff
@@ -0,0 +1,21 @@
+package application.moves;
+
+import application.Anature;
+import application.enums.MoveIds;
+
+public class Tackle extends Move
+{
+	private int mDamageDone;
+	
+	public Tackle()
+	{
+		super(25, MoveIds.Tackle, true);
+		mDamageDone = 30;
+	}
+
+	@Override
+	public void activateMove(Anature source, Anature target)
+	{
+		target.takeDamage(mDamageDone);
+	}
+}
```

Comment:
I see you needed an ability for the demo. We should make a case and push it through the board.

Reply:
That's a move, not an ability, but sure I'll make a story real quick.